ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml

```diff
@@ -0,0 +1,224 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v0.16.0_knative-eventing-src:
+    name: knative-v0.16.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.16.0_knative-serving-src:
+    name: knative-v0.16.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.17.1_knative-eventing-contrib-src:
+    name: knative-v0.17.1
+    namespace: openshift
+    tag: knative-eventing-contrib-src
+  openshift_knative-v0.17.2_knative-eventing-src:
+    name: knative-v0.17.2
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.17.3_knative-serving-src:
+    name: knative-v0.17.3
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.18.0_knative-eventing-contrib-src:
+    name: knative-v0.18.0
+    namespace: openshift
+    tag: knative-eventing-contrib-src
+  openshift_knative-v0.18.2_knative-serving-src:
+    name: knative-v0.18.2
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.18.4_knative-eventing-src:
+    name: knative-v0.18.4
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.18.6_knative-eventing-src:
+    name: knative-v0.18.6
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.19.0_knative-serving-src:
+    name: knative-v0.19.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.19.1_knative-eventing-kafka-src:
+    name: knative-v0.19.1
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_knative-v0.19.2_knative-eventing-src:
+    name: knative-v0.19.2
+    namespace: openshift
+    tag: knative-eventing-src
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.19.0_knative-serving-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.19.0:knative-serving-src
```

Comment:
@mgencur I believe these image references should be update to `v0.20.0` and added to base images section as well.

Reply:
This is done automatically. And it was already done here one hour ago: https://github.com/openshift/release/commit/6a37b42bc822f5add417d26236248586b1556b79

Reply:
That's neat and good to know, thanks! :)